projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js

```diff
@@ -1,12 +1,9 @@
 const path = require('path');
 // @TODO update require. when we export this method - we can connect it from index.js
-const {
-  parseDirectoryFiles,
-  getHeaders
-} = require('../../../../dist/index.cjs');
+const { parseDirectoryFiles, parseCsv } = require('../../../../dist/index.cjs');
 
 async function Main() {
-  const headers = await getHeaders('Fish_Reftbl_RefDatasets.csv');
+  const headers = await parseCsv(`${__dirname}/Fish_Reftbl_RefDatasets.csv`);
```

Comment:
if now we don't use getHeaders in a streight way - maybe it better to remove it's export at index.js?

Reply:
another request - can you also make same change at other parsers, located at FAO folder?